Assets/MixedRealityToolkit/_Core/Inspectors/CanvasEditorExtension.cs

```diff
@@ -27,17 +27,27 @@ private void OnEnable()
         {
             canvas = (Canvas)target;
 
-            if (MixedRealityManager.ConfirmInitialized())
+            if (MixedRealityManager.IsInitialized &&
+                MixedRealityManager.ConfirmInitialized() &&
+                MixedRealityManager.Instance.ActiveProfile.EnableInputSystem)
             {
                 inputSystem = MixedRealityManager.Instance.GetManager<IMixedRealityInputSystem>();
+                CheckCanvasSettings();
             }
         }
 
         public override void OnInspectorGUI()
         {
-            if (inputSystem == null)
+            if (!MixedRealityManager.IsInitialized || !MixedRealityManager.Instance.ActiveProfile.EnableInputSystem)
```

Comment:
What if the ActiveProfile is null?
Shouldn't happen, but if the user deletes their current profile, it will be null